ip.go

```diff
@@ -28,57 +29,109 @@ func (exo *Client) AssociateIpAddress(profile IpAddressProfile, async AsyncInfo)
 }
 
 // DestroyIpAddress is an alias for DisassociateIpAddress
-func (exo *Client) DestroyIpAddress(ipAddressId string, async AsyncInfo) (bool, error) {
+func (exo *Client) DestroyIpAddress(ipAddressId string, async AsyncInfo) error {
 	return exo.DisassociateIpAddress(ipAddressId, async)
 }
 
 // DisassociateIpAddress disassociates a public IP from the account
-func (exo *Client) DisassociateIpAddress(ipAddressId string, async AsyncInfo) (bool, error) {
+func (exo *Client) DisassociateIpAddress(ipAddressId string, async AsyncInfo) error {
 	params := url.Values{}
 	params.Set("id", ipAddressId)
 	resp, err := exo.AsyncRequest("disassociateIpAddress", params, async)
 	if err != nil {
-		return false, err
+		return err
 	}
 
 	var r BooleanResponse
 	if err := json.Unmarshal(resp, &r); err != nil {
-		return false, err
+		return err
 	}
 
-	return r.Success, nil
+	if !r.Success {
+		return fmt.Errorf("Cannot disassociateIpAddress. %s", r.DisplayText)
+	}
+
+	return nil
+}
+
+// GetAllIpAddresses returns all the public IP addresses
+func (exo *Client) GetAllIpAddresses() ([]*IpAddress, error) {
```

Comment:
Will be useful to have another shortcut to list all IPs in a zone.

Reply:
yagni!